vcpkg/concepts/features.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Understanding features in vcpkg
+description: This article explains the concept of features, their capabilities, and their expected contents in detail.
+author: JavierMatosD
+ms.author: javiermat
+ms.date: 02/21/2024
+ms.topic: concept-article
+---
+
+# Concept: features
+
+## Features
+
+Manifests can define additive sets of functionality, behavior, and dependencies called _features_.
```

Comment:
```suggestion
Ports can define additive sets of functionality, behavior, and dependencies called _features_.
```

While they are defined in port manifests, we should avoid making it sound like a manifest exclusive feature.